release_notes/topics/15_1_0.adoc

```diff
@@ -0,0 +1,39 @@
+= Highlights
+
+== Quarkus distribution
+
+Without comparison the biggest highlight of this release is all the improvements that have been made to the Quarkus distribution. So, many in fact that it will be hard to list them all.
+
+The CLI has been polished to hell and back, and we believe it now provides a very simple and convenient approach to configuring and running Keycloak. It's almost so simple that documentation shouldn't be needed.
```

Comment:
I would add a sentence along these lines: 

"To get started, just unpack the distribution, type ./kc.sh -h and discover the new possibilities!" 

as a call to action here. First thought about mentioning installing the autocomplete also, but it's in the hints of ./kc.sh -h :)